src/codelabs/dart-cheatsheet.md

```diff
@@ -1507,55 +1503,55 @@ class MyClass {
   final String aString;
   final double aDouble;
   
-  // Create a constructor here.
-}
-{$ end main.dart $}
-{$ begin solution.dart $}
-class MyClass {
-  final int anInt;
-  final String aString;
-  final double aDouble;
-  
-  MyClass(this.anInt, this.aString, this.aDouble);
+  // TODO: Create a constructor here.
```

Comment:
Should this be "the constructor" since we're instructed them to create a specific one?

```suggestion
  // TODO: Create the constructor here.
```